### PR TITLE
another round of dev-tools edits

### PR DESCRIPTION
**Pull request by chriddyp**
- explicitly handle the nested array case (`html.Div(children=[['oh no']])`)
- explicitly handle dicts as children: (`html.Div(children={'why': 'oh why'}`)
- a few css edits

### CODE REVIEW

**Comment by chriddyp** [comment]
dict case
image

nested lists
image

**Comment by chriddyp** [comment]
and here are the test cases that i'm working from: https://gist.github.com/chriddyp/93d5c12b88d7333f56457b867e94d7b6

File: src/TreeContainer.js [hunk 1]
```diff
@@ -25,6 +26,29 @@ import { assertPropTypes } from 'check-prop-types';
...
+            'of just a list. ' +
+            'Check the component that has the following contents, ' +
+            'and remove of the levels of nesting: \n' +
+            JSON.stringify(componentDefinition, null, 2)
+        );
+    }
+    if (type(componentDefinition) === 'Object' &&
+            !has('namespace', componentDefinition) &&
+            !has('type', componentDefinition) &&
+            !has('props', componentDefinition)) {
```
Comment (byronz) [inline src/TreeContainer.js]: I feel like the last two `&&` should be `||`

File: src/components/error/GlobalErrorOverlay.react.js [hunk 2]
```diff
@@ -14,14 +15,10 @@ export default class GlobalErrorOverlay extends Component {
...
-                errors = error.frontEnd;
-            }
-
-            error.backEnd.forEach(backEndError => {
-                errors.push(backEndError);
-            });
+            const errors = concat(
+                error.frontEnd,
+                error.backEnd
+            );
```
Comment (chriddyp) [inline src/components/error/GlobalErrorOverlay.react.js]: @byronz - this should fix 
image

Comment (byronz) [reply src/components/error/GlobalErrorOverlay.react.js]: good catch ! :+1:

**Pull request merged**